example/ext-pipe-plugin-example/pom.xml

```
@@ -21,16 +21,18 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.iotdb</groupId>
     <artifactId>iotdb-ext-pipe-example</artifactId>
+    <version>0.14.0-SNAPSHOT</version>
```

We should make it easy for users to package and register this example. Some changed is needed:
1. Change name to ext-pipe-example
2. Add to modules list in parent pom.xml
3. `mvn spotless:apply`
4. Add the necessary files in resources/META-INFO/services to enabled SPI.
```suggestion

    <parent>
        <groupId>org.apache.iotdb</groupId>
        <artifactId>iotdb-examples</artifactId>
        <version>0.14.0-SNAPSHOT</version>
        <relativePath>../pom.xml</relativePath>
    </parent>
```

---

Thank you.
I did below changes
1) Change artifactId to ext-pipe-example. 
2) Add the necessary files in resources/META-INFO/services to enabled SPI.
3) mvn spotless:apply

For left comments, I do not do change.  Because
1) This plugin dev example codes will not depend on IoTDB project but only depend on external-pipe-api, which is to facilitate customers' development.
2) We do not want customers to dev plugin using IoTDB project .pom.